dev/tools/generate-release-changelog.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+if [ $# != 1 ]; then
+  echo "Usage: $0 BRANCH"
+  exit
+fi
+
+branch=$1
+
+git checkout $branch
+git pull
+changelog_entries_with_title=$(ls doc/changelog/*/*.rst)
+changelog_entries_no_title=$(echo "$changelog_entries_with_title" | grep -v "00000-title.rst")
+git checkout master
+git pull
+for f in $changelog_entries_with_title; do
+  cat $f >> released.rst
```

Comment:
Why not `cat $changelog_entries_with_title # deliberately not quoted`? same for rm below

Reply:
My idea was that if there is an issue with one changelog entry not existing in the master branch, this would at least work until then. But given that there is no mechanism to warn in this case and bypass such failures, it doesn't help much.

Reply:
Cf. updated version.